src/components/AddCustModal.js

```diff
@@ -2,6 +2,8 @@ import React, { Component } from "react";
 import { Modal, Button, Row, Col, Form } from "react-bootstrap";
 import Snackbar from "@material-ui/core/SnackBar";
 import IconButton from "@material-ui/core/IconButton";
+
+//import {FbLoad} from "./components/FbLoad";
```

Comment:
I understand that it sometimes is ok to leave some out commented code in place for later reference, but you should try to clean out most of it!